src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/DomainEventEntity.kt

```diff
@@ -40,4 +44,11 @@ enum class DomainEventType {
   APPROVED_PREMISES_BOOKING_CANCELLED,
   APPROVED_PREMISES_BOOKING_CHANGED,
   APPROVED_PREMISES_APPLICATION_WITHDRAWN,
+  APPROVED_PREMISES_INFORMATION_REQUEST,
```

Comment:
I think we need a new enum for timeline event types, as we want them to sit apart from domain events, then we can have a mapping from one to the other.